modules/custom/social_graphql/src/Plugin/GraphQL/DataProducer/Payload/PayloadViolations.php

```diff
@@ -35,7 +35,7 @@ class PayloadViolations extends DataProducerPluginBase implements DataProducerPl
    * @return null|\Drupal\social_graphql\GraphQL\ViolationInterface[]
    *   The violations for this payload or null if there are none.
    */
-  public function resolve(PayloadInterface $payload) {
+  public function resolve(PayloadInterface $payload): ?ViolationInterface {
```

Comment:
This return type isn't correct I think. Looks like the comment return type also wasn't correct.